.eslintrc

```diff
@@ -150,7 +160,10 @@
     ],
     "lines-around-comment": [
       2,
-      { "afterLineComment": true, "allowBlockEnd": true }
+      {
+        "afterLineComment": false,
```

Comment:
I've changed this because the existing code wasn't following this rule (and also I think it's fairly common to have no empty line between a single line comment and the next line it explains)